packages/reassure-cli/src/commands/measure.ts

```diff
@@ -35,7 +35,11 @@ export function run(options: MeasureOptions) {
     { shell: true, stdio: 'inherit', env: { ...process.env, OUTPUT_FILE: outputFile } }
   );
 
-  console.log('');
```

Comment:
Not sure if this was intentional?

Reply:
This `console.log` was most probably inserting a new line there. Could you please restore it:
```ts
console.log(''); // insert newline
```